sandbox_starcheck

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 DIR=`dirname $0`
-export PYTHONHOME=$SKA
+PYTHON_EXE=`which python`
```

Comment:
I think there are some options for `which` in bash (maybe to skip aliases and such) but we probably don't need to care.

Reply:
The don't-care seems about right to me.